src/bootstrap.js

```diff
@@ -536,18 +546,21 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
      * @fires $"."connected
      */
     ChatEngine.connect = (uuid, state = {}, authKey) => {
-
-        // this creates a user known as Me and
-        // connects to the global chatroom
-        ChatEngine.pnConfig.uuid = uuid;
-        ChatEngine.pnConfig.authKey = authKey;
-
-        if (authKey) {
-            ChatEngine.handshake(() => {
+        if (!ChatEngine.isConnected) {
```

Comment:
@ManuelFernando this condition won't solve mentioned issue. 
For instance, user added some observers to _global_ and by mistake called `connect` one more time (after manually disconnected) - since this moment, _global_ on which he added observers won't receive any updates, because it will be replaced with new instance. 

In this case, we need property like in Chat: `hasConnected` [here](https://github.com/pubnub/chat-engine/blob/master/src/components/chat.js#L86). As you can see, it has been set only once in initial Chat connection (`connectionReady`) and used since then.

So, if you add this private property and check against it in this if clause, there won't be any troubles. 
Value of `hasConnected` should be set to **true** in `firstConnect` [here](https://github.com/pubnub/chat-engine/blob/master/src/bootstrap.js#L400) for instance.